Game/Resources/elevatorScene/Scripts/LightsOutScript.as

```diff
@@ -38,6 +42,8 @@ class LightsOutScript {
     }
 
     void Toggle(int index) {
+        buttonSound.Stop();
```

Comment:
Stop and Play only set flags. The flag set by Stop is immediately negated by Play, making the Stop call unnecessary.